graphql-server/pkg/queries/validator.go

```diff
@@ -46,26 +69,87 @@ func (q *ValidatorQuery) NewQuery(model interface{}, includeAddresses []string)
 		// Using model interface to avoid a bug where has-many fields are scanned as nil when using a nil model
 		Model(model).
 		Relation("Description").
+		Relation("VotingPower", func(votingPowerQuery *bun.SelectQuery) *bun.SelectQuery {
```

Comment:
let's open an issue to write test case for this query; too complicated to validate by inspection